app/src/main/java/com/paypal/psix/fragments/OnboardingFragment.java

```diff
@@ -0,0 +1,112 @@
+import com.paypal.psix.R;
+import com.paypal.psix.services.UserSession;
+
+import java.util.Arrays;
+import java.util.List;
+
+
+public class OnboardingFragment extends Fragment {
+
+    private static final String LOG_TAG = "MainFragment";
+    private static final List<String> PERMISSIONS = Arrays.asList("public_profile");
+
+    private LoginButton loginButton;
+    private UiLifecycleHelper uiHelper;
+    private GraphUser user;
+
+    private Session.StatusCallback callback = new Session.StatusCallback() {
+        @Override
+        public void call(Session session, SessionState state, Exception e) {
+            onSessionStateChange(session, state, e);
+        }
+    };
+
+    private FacebookDialog.Callback dialogCallback = new FacebookDialog.Callback() {
+        @Override
+        public void onComplete(FacebookDialog.PendingCall pendingCall, Bundle bundle) {
+            Log.d(LOG_TAG, "Success!");
+        }
+
+        @Override
+        public void onError(FacebookDialog.PendingCall pendingCall, Exception e, Bundle bundle) {
+            Log.d(LOG_TAG, String.format("Error: %s", e.toString()));
+        }
+    };
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.fragment_onboarding, container, false);
+        uiHelper = new UiLifecycleHelper(this.getActivity(), callback);
+        uiHelper.onCreate(savedInstanceState);
+
+        loginButton = (LoginButton) view.findViewById(R.id.authButton);
+        loginButton.setReadPermissions(PERMISSIONS);
+        loginButton.setFragment(this);
+        loginButton.setUserInfoChangedCallback(new LoginButton.UserInfoChangedCallback() {
+            @Override
+            public void onUserInfoFetched(GraphUser graphUser) {
+                user = graphUser;
+                updateUserData();
+            }
+        });
+
+        return view;
+    }
+
+    private void updateUserData() {
+        if (user != null) {
+            UserSession.setUser(user);
+        } else {
+            Log.d(LOG_TAG, "Current user is null");
+        }
+    }
+
+    @Override
+    public void onResume() {
+        super.onResume();
+        uiHelper.onResume();
+    }
+
+    @Override
+    public void onSaveInstanceState(Bundle outState) {
+        super.onSaveInstanceState(outState);
+        uiHelper.onSaveInstanceState(outState);
+    }
+
+    @Override
+    public void onActivityResult(int requestCode, int resultCode, Intent data) {
+        super.onActivityResult(requestCode, resultCode, data);
+        uiHelper.onActivityResult(requestCode, resultCode, data);
+    }
+
+    @Override
+    public void onPause() {
+        super.onPause();
+        uiHelper.onPause();
+    }
+
+    @Override
+    public void onDestroy() {
+        super.onDestroy();
+        uiHelper.onDestroy();
+    }
+
+    private void onSessionStateChange(Session session, SessionState state, Exception e) {}
```

Comment:
Copied from the FacebookHello example, and it seems it no longer is.

Removed :)